src/backend/base/langflow/alembic/versions/5ace73a7f223_new_remove_table_upgrade_op.py

```diff
@@ -0,0 +1,62 @@
+"""new remove table upgrade op
+
+Revision ID: 5ace73a7f223
+Revises: 0ae3a2674f32
+Create Date: 2024-10-08 10:59:12.980671
+
+"""
+
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+import sqlmodel
+from sqlalchemy.engine.reflection import Inspector
+from langflow.utils import migration
+from sqlalchemy.dialects import sqlite
+from langflow.utils import migration
+
+# revision identifiers, used by Alembic.
+revision: str = "5ace73a7f223"
+down_revision: Union[str, None] = "0ae3a2674f32"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    # ### commands auto generated by Alembic - please adjust! ###
+    if migration.table_exists("vertex_build", conn):
+        op.drop_table("vertex_build")
```

Comment:
```suggestion
```
I'm not sure why this is here but I think that is what is causing CI to fail

Reply:
If we remove this command then we get an error in alembic check because according to new model I believe we are removing vertex table.